stl/inc/xatomic.h

```diff
@@ -56,6 +56,15 @@ _STL_DISABLE_CLANG_WARNINGS
 _STD_BEGIN
 
 // ENUM memory_order

```

Comment:
This comment should be moved into the pre-C++20 half, and `// ENUM CLASS memory_order` should be added to the C++20 half, like: https://github.com/microsoft/STL/blob/da76ab2d5f4186f090ceaa07af4a3a2c070c749e/stl/inc/xcharconv.h#L28

Reply:
Fixed.